src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -275,11 +275,14 @@ async def coursier_resolve_lockfile(
                 [
                     coursier_report_file_name,
                     *(req.to_coord_str() for req in artifact_requirements),
-                    *(
-                        f"--strict-include={req.to_coord_str(versioned=False)}"
-                        for req in artifact_requirements
-                        if req.strict
-                    ),
+                    # TODO(#13496): Disable --strict-include to work around Coursier issue
```

Comment:
Should I pull this change out into its own PR?

Reply:
Fine with doing it here.